docker/entrypoint.sh

```diff
@@ -4,8 +4,8 @@ cat <<EOF > /etc/orchestrator.conf.json
 {
   "Debug": true,
   "ListenAddress": ":3000",
-  "MySQLTopologyUser": "${ORC_TOPOLOGY_USER:-:orchestrator}",
-  "MySQLTopologyPassword": "${ORC_TOPOLOGY_PASSWORD:-:orchestrator}",
```

Comment:
Because it would default to 
```
"MySQLTopologyUser": ":orchestrator",
"MySQLTopologyPassword": ":orchestrator"
```
instead of
```
"MySQLTopologyUser": "orchestrator",
"MySQLTopologyPassword": "orchestrator"
```
if `ORC_TOPOLOGY_USER` or `ORC_TOPOLOGY_PASSWORD` is not set (Notice the extra `:` in the original version).